src/main/java/com/traveltime/benchmarks/UriValidation.java

```diff
@@ -0,0 +1,27 @@
+package com.traveltime.benchmarks;
+
+import com.traveltime.sdk.TravelTimeSDK;
+import com.traveltime.sdk.dto.requests.TimeFilterFastProtoRequest;
+import com.traveltime.sdk.dto.requests.proto.OneToMany;
+import lombok.val;
+import static com.traveltime.benchmarks.BenchmarkSetup.*;
+
+
+public class UriValidation {
+
+    public static void correctApiUri(){
```

Comment:
I'd call it `validateBenchmarkSetup` or something, since it's not just the URI you're checking.